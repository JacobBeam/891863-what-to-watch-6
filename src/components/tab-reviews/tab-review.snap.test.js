import React from 'react';
import {render} from '@testing-library/react';
import {createMemoryHistory} from 'history';
import * as redux from 'react-redux';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import {Router} from 'react-router-dom';
import TabReviews from './tab-reviews';

const api = jest.fn();
const middlleware = thunk.withExtraArgument(api);
const mockStore = configureStore([middlleware]);


it(`Should TabReviews render correctly`, () => {
  const history = createMemoryHistory();
  const store = mockStore({
    COMMENT_DATA: {
      isSentComment: false,
      selectedComments: []
    }
  });

  const {container} = render(
      <redux.Provider store={store}>
        <Router history={history}>
          <TabReviews/>
        </Router>
      </redux.Provider>
  );
  expect(container).toMatchSnapshot();
});
