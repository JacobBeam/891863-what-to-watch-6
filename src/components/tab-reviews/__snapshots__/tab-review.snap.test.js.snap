// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TabReviews render correctly 1`] = `
<div>
  <div
    class="movie-card__reviews movie-card__row"
  >
    <div
      class="movie-card__reviews-col"
    />
    <div
      class="movie-card__reviews-col"
    />
  </div>
</div>
`;
